src/haxeLanguageServer/features/haxe/DocumentFormattingFeature.hx

```diff
@@ -69,11 +69,9 @@ class DocumentFormattingFeature {
 				} else {
 					range;
 				}
-				if (doc.getText(range) != formattedCode) {
-					final edits = [{range: range, newText: formattedCode}];
-					resolve(edits);
-					onResolve(null, edits.length + " changes");
-				}
+				final edits = doc.getText(range) != formattedCode ? [{range: range, newText: formattedCode}] : [];
```

Comment:
We don't use ternary in this code base, we use inline if-else expressions instead.

Reply:
Ok, np.